changelog.txt

```diff
@@ -3,6 +3,7 @@
 = 3.0.0 - 2021-xx-xx =
 * Add - Download deposits report in CSV.
 * Fix - Use store currency on analytics leaderboard when Multi-Currency is enabled.
+* Fix - Add enabled currencies modal UI.
```

Comment:
Nitpick here 🙂 I think might be better to describe the actual fix, cause looks like you're adding the modal for the first time. Maybe just removing the `Add ` is enough.

Reply:
Makes sense 🙂
Done in https://github.com/Automattic/woocommerce-payments/pull/2840/commits/aa25a0d07d680278e9a36645e6ca3b894bd53036